<launch> 

  <arg name="nao_name" default="$(optenv NAO_NAME nao)"/>
  <node pkg="drift_correction" type="drift_correction_node" name="drift_correction" output="screen">
    <param name="sensor_frame_id" value="marker" type="str"/>
    <param name="tracked_frame_id" value="$(arg nao_name)_marker/base_link" type="str"/>

    <param name="global_frame_id" value="map" type="str" />
    <param name="publishing_frequency" value="30" type="int"/>
    <param name="tf_prefix" value="$(arg nao_name)" />
  </node>

  <node pkg="tf" type="static_transform_publisher" name="head_to_sensor" args="0 0 0 0 1.57 0 $(arg nao_name)/HeadTouchMiddle_frame $(arg nao_name)/marker 10" />
</launch>
